docs/api/browser-window.md

```diff
@@ -1627,7 +1627,7 @@ Returns `Boolean` - Whether the menu bar is visible.
 * `visible` Boolean
 * `options` Object (optional)
   * `visibleOnFullScreen` Boolean (optional) _macOS_ - Sets whether
-    the window should be visible above fullscreen windows
+    the window should be visible above fullscreen windows _deprecated_
```

Comment:
I think we usually put `_Deprecated_` next to `_macOS_` etc.?

Reply:
for options (not top-level apis themselves) it seems to be a mix: https://github.com/electron/electron/blob/14cc902ad83a21c1dab7765166e9f3402cb48c3c/docs/api/system-preferences.md#L294

i'm impartial though, happy to move it!